packages/svelte/src/internal/client/runtime.js

```diff
@@ -257,9 +264,10 @@ export function check_dirtiness(reaction) {
 					// In thise case, we need to re-attach it to the graph and mark it dirty if any of its dependencies have
 					// changed since.
 					if (version > /** @type {import('#client').Derived} */ (reaction).version) {
-						/** @type {import('#client').Derived} */ (reaction).version = version;
+						/** @type {import('#client').Derived} */ (reaction).version = increment_version();
```

Comment:
Why are you incrementing here? You should be matching the value otherwise you might cause additional invalidations that aren’t necessary.

Reply:
i don't think it would cause unnecessary invalidations. but yeah, we actually only need to increment in two places — in `set` and `update_derived`